index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+
+
+</head>
+<body>
+    <nav class="navigation">
+       <img src="./logos/MyLogo.png" alt="navigation icon" class="navigation__logo">
+       <img src="./logos/menu.svg" alt="navigation icon" class="navigation__hamburger">
+    </nav>
+    
+<div class="bg">
```

Comment:
- [ ] You should use a `<section>` tag instead of `<div>` here to wrap the headline section for semantic reasons.